requirements.txt

```diff
@@ -1,3 +1,5 @@
 pyfrc>=2018.0.0
 robotpy-ctre>=2018.0.0
 pynetworktables>=2018.0.0
+pybind11>=2.2.2
```

Comment:
Our favorite!

Reply:
Did this give you trouble in the past?

Reply:
I believe we forgot to install it on the RIO last year and it took us a while to figure out what we'd done wrong.